spec/models/has_tag_string_tag_spec.rb

```diff
@@ -94,7 +94,7 @@ def self.down
       let!(:model_3) { ModelWithTag.create(name: 'a', tag_string: 'foo test') }
 
       it 'sorts by name ASC' do
-        expect(subject).to match([model_3, model_1, model_2])
+        expect(subject).to match_array([model_3, model_1, model_2])
```

Comment:
Just a quick early review – This looks like a case where we _do_ care about order, so I don't think a global find and replace is what we want to do.

Reply:
Ah okay, will revert that change